modules/ROOT/pages/getting-started-with-api-experience-hub.adoc

```diff
@@ -1,14 +1,15 @@
-= Prerequisites 
+= Getting Started with API Experience Hub 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: prerequisites.adoc
 
 Verify that you have the following required accounts and access:
 
 * Anypoint Platform organization in production
 * Salesforce organization
 * Organization Administrator of the main organization
-* You must be a System Administrator in the Salesforce organization that you are linking to during the connection process
+* You must have the System Administrator profile or a Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process. For more information, see xref:assigning-permissions.adoc[].
```

Comment:
```suggestion
* You must have the System Administrator profile or be a Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process. For more information, see xref:assigning-permissions.adoc[].
```

Reply:
Not sure if that's correct.

Reply:
Does a profile get assigned? If so, then maybe something like:
* You must be assigned either the System Administrator profile ...

Reply:
This is hard to parse. I wonder if a sublist is in order.

You must either be:
* Assigned the System Administrator profile
* A Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process.